config/swagger.yaml

```diff
@@ -2467,181 +2467,51 @@ definitions:
   SpendTxJSON:
     allOf:
       - $ref: '#/definitions/JSONTx'
-      - type: object
-        properties:
-          sender: string
-          recipient: string
-          amount: integer
-          fee: integer
-          nonce: integer
-        required:
-        - sender
-        - recipient
-        - amount
-        - fee
-        - nonce
+      - $ref: '#/definitions/SpendTx'
```

Comment:
Re this redundancy - I see the def SpendTx (I did not check the other ones) is more refined. Maybe this redundancy was in place because swagger-codegen was not able to generate the Python client - or the generated Python client was misbehaving? I am concerned that UAT tests may not work.

Reply:
Ran tests locally and they've passed